specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimanagement.json

```diff
@@ -0,0 +1,484 @@
+        "tags": [
+          "Policy"
+        ],
+        "operationId": "Policy_CreateOrUpdate",
+        "description": "Creates or updates the global policy configuration of the Api Management service.",
+        "x-ms-examples": {
+          "ApiManagementPutPolicies": {
+            "$ref": "./examples/ApiManagementPutPolicies.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PolicyIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/PolicyContract"
+            },
+            "required": true,
+            "description": "The policy contents to apply."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Global policy configuration was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/PolicyContract"
+            }
+          },
+          "200": {
+            "description": "Global policy configuration of the Api Management service was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/PolicyContract"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Policy"
+        ],
+        "operationId": "Policy_Delete",
+        "description": "Deletes the global policy configuration of the Api Management Service.",
+        "x-ms-examples": {
+          "ApiManagementDeletePolicies": {
+            "$ref": "./examples/ApiManagementDeletePolicies.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PolicyIdParameter"
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": true,
+            "description": "The entity state (Etag) version of the policy to be deleted. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Successfully deleted the policy configuration at the Global level."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/policySnippets": {
+      "get": {
+        "tags": [
+          "PolicySnippets"
+        ],
+        "operationId": "PolicySnippets_List",
```

Comment:
>PolicySnippets_List [](start = 24, length = 19)

Could be marked as [pageable](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable)

Reply:
fixed